zcl-builtin/matter/data-model/chip/matter-devices.xml

```diff
@@ -2042,4 +2042,60 @@ limitations under the License.
             </include>            
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>MA-refrigerator</name>
+        <domain>CHIP</domain>
+        <typeName>Matter Refrigerator</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x0070</deviceId>
+        <class>Simple</class>
+        <scope>Endpoint</scope>
+        <endpointComposition>
+            <type> tree </type>
```

Comment:
Do we need another device type with endpoint composition full family?

Reply:
Sure but the logic is the same